spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+describe Person do
+  person = Person.new(20, name: 'Mark')
+
+  context 'should create instance of Person Class' do
+    it 'returns correct name' do
+      expect(person.name).to eq('Mark')
+    end
+
+    it 'returns correct age' do
+      expect(person.age).to eq(20)
+    end
+
+    it 'should return correct data' do
+      expect(person.can_use_services?).to eq true
+    end
+  end
+end
```

Comment:
- [ ] Thank you for your hard work on this PR! 🙌 It's looking great, and I can see the effort you've put into it. 🌟Before we merge this PR, let's make sure all the methods in your classes get tested! 🧐 Please take a moment to thoroughly test each method to ensure everything is working smoothly. 🛠️. In this `Person class` the  `rent_book` method is not tested.